packages/cli/rollup.config.js

```diff
@@ -0,0 +1,41 @@
+import esbuild from 'rollup-plugin-esbuild'
```

Comment:
@lascenify this aims to mitigate the ES modules issues that we experienced. [RollupJS](https://rollupjs.org/guide/en/) is a modern Javascript bundler that understands both, ES modules and CommonJS, and outputs Javascript that adheres to a specific module system. The great thing about RollupJS is that the tool is very flexible and has a very powerful plugin system that we can make use of. I'll add some comments along this document explaining a few things.